### DISCUSSION TITLE
有没有人今天遇到频繁的发不出消息的问题

### DISCUSSION THREAD

**Discussion by Arondight** [takayama-lily/oicq#385] [Q&A] [answered]
提升什么什么太频繁还是啥来着，群消息发不出

![6A44357E-E408-4508-861D-CACA57EDC039](https://user-images.githubusercontent.com/13673702/178652526-6f161edc-2751-456a-b2b7-67300da09e01.png)

**Comment by takayama-lily** [comment, marked as answer]
https://github.com/mamoe/mirai/issues/2127

**Reply by Arondight** [reply]
真是个悲伤的故事
